imagedephi/templates/HomePage.html.j2

```diff
@@ -0,0 +1,125 @@
+          rel="stylesheet">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            "primary": "#5A387C",
+
+            "secondary": "#00A6BF",
+
+            "accent": "#FF6A6A",
+
+            "neutral": "#201C35",
+
+            "base-100": "#FFFFFF",
+
+            "info": "#3ABFF8",
+
+            "success": "#36D399",
+
+            "warning": "#FBBD23",
+
+            "error": "#F87272",
+
+            "secondaryContent": "#E8F2F3",
+          },
+        },
+      },
+      daisyui: {
+        themes: [
+          {
+            light: {
+
+              "primary": "#5A387C",
+
+              "secondary": "#00A6BF",
+
+              "accent": "#FF6A6A",
+
+              "neutral": "#201C35",
+
+              "base-100": "#FFFFFF",
+
+              "info": "#3ABFF8",
+
+              "success": "#36D399",
+
+              "warning": "#FBBD23",
+
+              "error": "#F87272",
+
+              "secondaryContent": "#E8F2F3",
+
+            },
+          },
+        ],
+      },
+    }
+    </script>
+  </head>
+  <body class="bg-base-300 min-h-screen">
+    <div class="flex">
+      <form method="post" action="{{ url_for("redact") }}">
+        <input type="hidden"
+               value="{{ input_directory_data.directory }}"
+               name="input_directory" />
+        <input type="hidden"
+               value="{{ output_directory_data.directory }}"
+               name="output_directory" />
+        <input type="checkbox" class="drawer-toggle" id="side-drawer" />
+        <div class="flex max-w-md">
+          <div class="pl-4 py-4 {{ 'opacity-50' if redacted==True  }}">
+            <div class="bg-base-100 drop-shadow-xl rounded flex flex-col">
+              <div class="flex justify-between content-center p-4 border-b">
+                <div class="max-h6 w-auto self-center">
+                  <img src="{{ url_for('assets', path='logo.png') }}" />
+                </div>
+                <div class="flex items-center space-y-0.5">
+                  <a class="btn btn-ghost btn-square btn-sm">
+                    <i class="ri-side-bar-line text-lg text-neutral"></i>
+                  </a>
+                </div>
+              </div>
+              {% include "SidebarInputStep.html.j2" %}
+              {% include "SidebarOutputStep.html.j2" %}
+              {% include "SidebarRulesetStep.html.j2" %}
+              <button type="submit" class="btn btn-wide bg-accent m-auto" {{ 'disabled' if redacted}}>De-phi
+                images
+              </button>
+            </div>
+          </div>
+        </div>
+      </form>
+      {# TODO make into a separate component #}
+      {% if redacted %}
+        <div class="px-8 py-4">
+          <div class="hero bg-base-200">
+            <div class="hero-content text-center">
+              <div>
+                <h1 class="text-5xl font-bold">Redaction Succesful!</h1>
```

Comment:
Succesful -> Successful

And, I always felt that exclamation points make it sound like we are surprised it actually happened.